src/sql/src/pure.rs

```diff
@@ -396,12 +396,23 @@ async fn purify_csr_connector_avro(
                 .expect("CSR Connector must specify Registry URL"),
         }
         .parse()?;
+        let mut ccsr_client_options =
+            if let CsrConnector::Reference { with_options, .. } = connector {
+                with_options
+                    .as_ref()
+                    .expect("")
```

Comment:
`.expect("")` is just `unwrap()`

Reply:
Oops, this should've had a meaningful error string in it.